src/main/java/core/basesyntax/Application.java

```diff
@@ -2,6 +2,12 @@
 
 public class Application {
```

Comment:
The `Application` class should have an access modifier specified. It's best practice to include access modifiers to clearly define the scope of a class.